src/command.rs

```diff
@@ -244,6 +244,7 @@ pub async fn install() -> miette::Result<()> {
     let store = PackageStore::current().await?;
     let credentials = Credentials::load().await?;
 
+    store.clear().await?;
```

Comment:
Shouldn't this fail linting? There's now an unused `Option`.

I think the intent would be clearer if we did
```
let _ = store.clear().await;
```

Better yet, let's not silence errors and print a brief message about it, something like:
```
if let Err(err) = store.clear().await {
    warn!("failed to clear store: ", ?err);
}
```
Unless this is very common and would just be noise.

Reply:
There is no unused option?